README.md

```diff
@@ -89,6 +89,7 @@ Module managed by [Marcin Cuber](https://github.com/marcincuber) [linkedin](http
 | description | The description of the all resources. | `string` | `"Managed by Terraform"` | no |
 | engine\_version | The version number of the cache engine to be used for the cache clusters in this replication group. | `string` | `"5.0.6"` | no |
 | family | The family of the ElastiCache parameter group. | `string` | `"redis5.0"` | no |
+| ingress\_self | Specify whether the security group itself will be added as a source to the ingress rule. | `bool` | `true` | no |
```

Comment:
Update docs accordingly as default will be set to false.

Reply:
Fixed. Thanks.